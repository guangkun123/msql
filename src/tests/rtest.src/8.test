# Do a sorted join

select test2.fullname, test.age
        from test2,test
        where test2.user = test.user
        order by test2.fullname
\p\g

