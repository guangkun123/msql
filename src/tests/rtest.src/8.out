
Welcome to the miniSQL monitor.  Type \h for help.


mSQL >     ->     ->     ->     ->     -> 
Query buffer
------------

select test2.fullname, test.age
        from test2,test
        where test2.user = test.user
        order by test2.fullname

[continue]
    -> 
Query OK.

3 rows matched.

 +--------------------------------+----------+
 | fullname                       | age      |
 +--------------------------------+----------+
 | David J. Hughes                | 26       |
 | Fred O'Niell                   | 31       |
 | Paul Pyyvaara                  | 26       |
 +--------------------------------+----------+



mSQL >     -> 
Bye!

